pkg/api/k8s_versions.go

```diff
@@ -45,8 +45,8 @@ const (
 	csiSnapshotControllerImageReference               string = "oss/kubernetes-csi/snapshot-controller:v2.0.0"
 	csiAzureDiskImageReference                        string = "k8s/csi/azuredisk-csi:v0.7.0"
 	csiAzureFileImageReference                        string = "k8s/csi/azurefile-csi:v0.6.0"
-	azureCloudControllerManagerImageReference         string = "oss/kubernetes/azure-cloud-controller-manager:v0.5.1"
-	azureCloudNodeManagerImageReference               string = "oss/kubernetes/azure-cloud-node-manager:v0.5.1"
+	azureCloudControllerManagerImageReference         string = "oss/kubernetes/azure-cloud-controller-manager:v1.1.0"
```

Comment:
is it OK to update this for all clouds?

Reply:
The change works for Azure and Azure Stack. For other cloud, I think they can continue to use with "useCloudControllerManager" flag set to false, or adopt similar change we did if they want to use external cloud provider as well. Does this make sense?